src/st/dre/1A3A8.c

```diff
@@ -37,25 +37,9 @@ u8 func_8019A590(u8 frames[], Entity* self, u8 arg2) {
     return var_a1;
 }
 
-// Absolute distance from g_CurrentEntity to the player in the X Axis
-s32 GetDistanceToPlayerX(void) {
-    s16 xDistance = g_CurrentEntity->posX.i.hi - PLAYER.posX.i.hi;
+#include "../get_distance_to_player_x.h"
 
-    if (xDistance < 0) {
-        xDistance = -xDistance;
-    }
-    return xDistance;
-}
-
-// Absolute distance from g_CurrentEntity to the player in the Y Axis
-s32 GetDistanceToPlayerY(void) {
-    s32 yDistance = g_CurrentEntity->posY.i.hi - PLAYER.posY.i.hi;
-
-    if (yDistance < 0) {
-        yDistance = -yDistance;
-    }
-    return yDistance;
-}
+#include "../get_distance_to_player_y.h"
 
 s32 GetSideToPlayer(void) {
```

Comment:
I know it is not part of your change, but do you mind to also modify this as `#include "../get_side_to_player.h"`?